presto-hive-common/src/main/java/com/facebook/presto/hive/zorder/ZAddressRange.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.zorder;
+
+import java.util.Objects;
+
+public class ZAddressRange
+{
+    private long minAddress;
+    private long maxAddress;
+
+    public ZAddressRange(long minAddress, long maxAddress)
+    {
+        this.minAddress = minAddress;
+        this.maxAddress = maxAddress;
+    }
+
+    public long getMinAddress()
+    {
+        return minAddress;
+    }
+
+    public long getMaxAddress()
+    {
+        return maxAddress;
+    }
+
+    public void setMaxAddress(long maxAddress)
```

Comment:
Given this class is more like a wrapper, do we need a setter? How about making this class immutable and create new object when needed?